src/main/java/com/gildedrose/items/DefaultItem.java

```diff
@@ -7,18 +7,28 @@ public class DefaultItem implements Ageable {
     private final Item item;
     private final int minQuality;
     protected int qualityCoefficient;
+    private final float markUp;
 
     public DefaultItem(Item item) {
         this.item = item;
         this.qualityCoefficient = 1;
         this.minQuality = 0;
+        this.markUp = 1.3f;
```

Comment:
This markup is repeated in other items which means when we add a new item, we have to remember to add this attribute. Is there a way we can declare this markup value once and use it everywhere it's relevant?

Reply:
This is a good point and is addressed in [this commit](https://github.com/jramnani/GildedRoseJavaLLC/pull/24/commits/4d14973464d91e4b3f54b792d99e9a23a1f9b55c)!